src/_codux/boards/hats/hats.board.tsx

```diff
@@ -0,0 +1,14 @@
+import { createBoard } from '@wixc3/react-board';
+import Hats from '../../../pages/hats';
+
+export default createBoard({
+   name: 'Hats',
+   Board: () => <Hats />,
+   isSnippet: true,
+   environmentProps: {
+      canvasWidth: 2098,
+      windowWidth: 1332,
+      windowHeight: 947,
+      canvasHeight: 1516,
```

Comment:
Notice these dimensions are changed and saved when you resize both window and canvas.
So it looks a bit misplaced when the dimensions are as such.